### PR TITLE
Add Presses universitaires de Strasbourg style

### PR DESCRIPTION
**Pull request by etnadji**
new file:   pus-note-fr.csl

According to [current instructions to authors](http://pus.unistra.fr/html/WYSIWYGfiles/files/Note-aux-auteurs_finale.pdf).

EDIT : Fixed conflicting ID -> pus-note-fr

### CODE REVIEW

File: pus-note-fr.csl [hunk 1]
```diff
@@ -0,0 +1,711 @@
...
+      <term name="available at">disponible à l'adresse</term>
+      <term name="editor" form="short">dir.</term>
+      <term name="cited">, cité</term>
+      <term name="translator" form="verb-short">traduction de</term>
+      <term name="translator" form="verb">traduction de</term>
+      <term name="no date" form="short"></term>
+      <!-- Termes non-standards -->
+      <term name="opcit">op.&#160;cit.</term>
+      <term name="artcit">art.cité</term>
+      <term name="doiacronym">DOI&#160;:&#160;</term>
```
Comment (rmzelle) [inline pus-note-fr.csl]: @etnadji, the style won't validate with non-standard CSL terms. You can use prefixes or verbatim text elements instead. E.g. replace

```xml
  <macro name="doi">
    <text term="doiacronym"/>
    <text variable="DOI"/>
  </macro>
```

by

```xml
  <macro name="doi">
    <text prefix="DOI&#160;:&#160;" variable="DOI"/>
  </macro>
```

or

 ```xml
  <macro name="doi">
    <group delimiter="&#160;:&#160;">
      <text value="DOI"/>
      <text variable="DOI"/>
    </group>
  </macro>
```

File: pus-note-fr.csl [hunk 2]
```diff
@@ -0,0 +1,711 @@
...
+            <if variable="medium">
+              <text variable="medium"/>
+            </if>
+          </choose>
+          <text macro="online"/>
+        </group>
+      </else-if>
+      <else>
+        <!-- <text macro="online" prefix="[" suffix="]"/> -->
+      </else>
```
Comment (rmzelle) [inline pus-note-fr.csl]: Empty `<else/>` elements are also not allowed, and should just be deleted.

File: pus-note-fr.csl (hunk 1 above)
Comment (etnadji) [reply pus-note-fr.csl]: Thanks for this information ! It worked fine when I used it, maybe the CSL formatter was the one Zotero use.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**